src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,44 +1,66 @@
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { TodosContext } from '../../TodosContext';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
 
 export const TodoModal: React.FC = () => {
+  const { selectedTodo, setSelectedTodo } = useContext(TodosContext);
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    if (!selectedTodo) {
+      return;
+    }
+
+    setIsLoading(true);
+    getUser(selectedTodo?.userId)
```

Comment:
add error handling

https://github.com/mate-academy/react_dynamic-list-of-todos/assets/53228259/b9c7a5d9-3465-4878-9bb9-89a7acb6e153